elasticdl/python/master/start_redis.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
```

Comment:
The module name `start_redis` seems a little weird.

Reply:
Done